lib/widgets/game_view.dart

```diff
@@ -15,8 +15,8 @@ class GameView extends StatefulWidget {
 }
 
 class _GameViewState extends State<GameView> {
-  int whoWin = 0;
-  int inRowValu = 3;
+  bool rowBuilderWins = false;
```

Comment:
Let it default to true, a traditional game.

Reply:
Done